lib/app_web/templates/person/info.html.eex

```diff
@@ -2,6 +2,8 @@
   <img class="br-100 fl " width="32px" src="<%= @person.picture %>" />
 </a>
 
+<a class="db tr pointer dwyl-teal" href="<%= Routes.person_path(@conn, :logout) %>">Log out</a>
```

Comment:
Apparently `eex` templates aren't checked for length by `mix format` ... 95 characters. 💭